src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -866,6 +867,51 @@ class spell_warr_second_wind_trigger : public SpellScriptLoader
         }
 };
 
+// Shield Wall - 871
+class spell_warr_shield_wall : public SpellScriptLoader
+{
+public:
+    spell_warr_shield_wall() : SpellScriptLoader("spell_warr_shield_wall") { }
+
+    class spell_warr_shield_wall_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_warr_shield_wall_AuraScript);
+
+        void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+        {
+            if (!GetCaster())
+                return;
+
+            if (Player* player = GetCaster()->ToPlayer())
+            {
+                player->AddAura(SPELL_WARRIOR_SHIELD_OF_WALL, player);   
+            }
+        }
+
+        void OnRemove(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+        {
+            if (!GetCaster())
+                return;
+
+            if (Player* player = GetCaster()->ToPlayer())
+            {
+                player->RemoveAura(SPELL_WARRIOR_SHIELD_OF_WALL);
```

Comment:
@HannibalRoG : Please rename this enum to `SPELL_WARRIOR_SHIELD_WALL`
The spell is named "Shield Wall", not Shield Of Wall.

@Keader : you may want to repost your comments after it has been changed.